File: ios/Classes/AmapOptionsSink.swift
//
//  AmapOptionsSink.swift
//  amap_view
//
//  Created by Spicely on 2020/7/6.
//


protocol AmapOptionsSink {
    func setCamera(camera: CameraPosition)
    func setCompassEnabled(compassEnabled: Bool)
    func setMapType(mapType: Int)
    func setRotateGesturesEnabled(rotateGesturesEnabled: Bool)
    func setMyLocationEnabled(myLocationEnabled: Bool)
    func setScaleEnabled(scaleEnabled: Bool)
    func setScrollEnabled(scrollEnabled: Bool)
    func setZoomEnabled(zoomEnabled: Bool)
    func setIndoorMap(indoorEnabled: Bool)
}

File: android/src/main/kotlin/com/muka/amap_view/AmapOptionsSink.kt
package com.muka.amap_view

interface AmapOptionsSink {
    // fun setCamera(camera: CameraPosition)
    fun setCompassEnabled(compassEnabled: Boolean)
    fun setMapType(mapType: Int)
    fun setRotateGesturesEnabled(rotateGesturesEnabled: Boolean)
    fun setScrollGesturesEnabled(scrollGesturesEnabled: Boolean)
    fun setScaleEnabled(scaleEnabled: Boolean)
    fun setZoomGesturesEnabled(zoomGesturesEnabled: Boolean)
    fun setZoomControlsEnabled(zoomControlsEnabled: Boolean)
    fun setMyLocationEnabled(myLocationEnabled: Boolean)
}